scripts/globals/treasure.lua

```diff
@@ -603,10 +603,13 @@ xi.treasure.treasureInfo =
                 misc =
                 {
                     {
-                        test = function(player) return player:getCharVar("needs_crawler_blood") == 1 end,
+                        test = function(player)
+                            return player:getCharVar("Quest[0][85]prog") >= 2 and
```

Comment:
Since we're not space-constrained (like in bcnm.lua), could you please use xi.quest.getVar and its parameters?

Reply:
Done